RELEASE_NOTES.md

```
@@ -1,45 +1,216 @@
+A big thanks to René Zubcevic and Àngel Ollé Blázquez for keeping the project alive this last year, and hopefully, we
+can make
+many more releases this year.
+
+### New functionality
+
+- New year's resolution(2022): major refactoring of WebGoat to simplify the setup and improve building times.
 - Move away from multi-project setup:
-  - This has a huge performance benefit when building the application. Build time locally is now `Total time:  42.469 s` (depends on your local machine of course)
-  - No longer add Maven dependencies in several places
-  - H2 no longer needs to run as separate process, which solves the issue of WebWolf sharing and needing to configure the correct database connection.
+  * This has a huge performance benefit when building the application. Build time locally is now
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)